docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ontotext/graphdb:10.5.1
```

Comment:
Thank you!

The docker-compose is supposed to support different services. 

Could we add a namespace docker/graphdb/ and place the Dockerfile within it as well as the grapdb_create.sh script?

Reply:
Ok, I have added them.